types/iframe-resizer/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for iframe-resizer 3.5
+// Type definitions for iframe-resizer 4.3
```

Comment:
I think this cannot be just updated, as the 4.0.0 seems to be breaking:
https://github.com/davidjbradshaw/iframe-resizer/blob/a519ec84ad4efb8eaadf40fc072b5e523ffe45ac/docs/upgrade.md#upgrading-to-version-4
https://github.com/davidjbradshaw/iframe-resizer/commit/da2e08f11ed6a005f81a027007972a7f35aca7ef#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed
We cannot ship this types to be used for v4, only for v3, unless api is migrated

Reply:
Is the issue here that some people will continue to rely on v3 of the original library, and therefore v3 of the typings? I'm not sure I understand why we can't ship types for v4.

Reply:
the api changed, so 'onClose' listener (if I recall correctly the name) no longer exists at runtime. if we ship as v4, without api change, somene will use this and code will fail at runtime, just because of DT types

Reply:
Ah, I got it now, thanks.